### PR TITLE
Add EJS icon

### LINKED ISSUE #6132: Create EJS Icon
**Brand Name: EJS**
**Website: https://ejs.co/**
**Alexa rank:116,305**
  

**Official resources for icon and color:**
  

Create an [EJS](https://ejs.co/) Icon please.

### PR DESCRIPTION
**Pull request by PratyushJayachandran**
ejs

**Issue:** closes #6132 

**GitHub Stars:** 7.2k. See https://github.com/mde/ejs

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG: Taken from https://ejs.co/favicon.svg
Color: Taken from logo `#a91e50`

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -3690,6 +3690,11 @@
             "hex": "262729",
             "source": "https://www.eightsleep.com/press/"
         },
+        {
+            "title": "EJS",
+            "hex": "A91E50",
+            "source": "https://ejs.co/favicon.svg"
```
Comment (adamrusted) [inline _data/simple-icons.json:3696]:
```suggestion
            "source": "https://github.com/mde/ejs-site/blob/dd845093a46b373df42108b888a0ad80085a5c94/ejs.svg"
```

There's a higher resolution version of the logo on GitHub that we can use here - could you update using that as your source?

**Review by adamrusted — CHANGES_REQUESTED** [review]

**Comment by PeterShaggyNoble** [comment]
Don't forget about this one, @PratyushJayachandran 😉 

Although it looks like [development might have stalled](https://github.com/mde/ejs/graphs/commit-activity) on the project.

**Comment by PratyushJayachandran** [comment]
> Don't forget about this one, @PratyushJayachandran 😉
> 
> Although it looks like [development might have stalled](https://github.com/mde/ejs/graphs/commit-activity) on the project.

Oops... missed this. I have updated the icon based on SVG from GitHub repository

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
SVG looks good to me, @PratyushJayachandran 👍🏻 But, from [their website](https://ejs.co/), it looks to me like their primary colour is the green and that the pink is the secondary.

Also, my concerns about stalled development above aside, I think we name definitely say [12.5m weekly downloads](https://www.npmjs.com/package/ejs) on npm makes it popular enough for us to include! 😆

**Review by PeterShaggyNoble — APPROVED** [review]

**Pull request merged**